SCORER.md

```diff
@@ -21,11 +25,231 @@ You may need to build your own scorer - your own _language model_ if:
 * You are training a speech recognition model for a particular domain - such as technical words, medical transcription, agricultural terms and so on
 * If you want to improve the accuracy of transcription
 
-DeepSpeech supports the _optional_ use of an external scorer - if you're not sure if you need to build your own scorer, stick with the built-in one to begin with.
+**DeepSpeech supports the _optional_ use of an external scorer - if you're not sure if you need to build your own scorer, stick with the built-in one to begin with**.
 
 ### Building your own scorer
 
-Building your own scorer is beyond the scope of the DeepSpeech Playbook, but the [DeepSpeech documentation covers how to do this](https://deepspeech.readthedocs.io/en/latest/Scorer.html). There are built-in scripts with DeepSpeech which make building an external scorer - a _language model_ - easier.
+_This section assumes that you are using a Docker image and container for training, as outlined in the [environment](ENVIRONMENT.md) section. If you are not using the Docker image, then some of the scripts such as `generate_lm.py` will not be available in your environment._
+
+_This section also assumes that you have already [trained a model](TRAINING.md) and have stored checkpoints for the trained model. This is required to generate the `--default_alpha` and `--default_beta` parameters that are used by the `lm_optimizer.py` script._
```

Comment:
err, `lm_optimizer.py` runs so we can know those default alpha/beta, the wording here seems to imply the contrary

Reply:
Great pickup , thank you.